### PR TITLE
Remove BlazeClient from dependencies

### LINKED ISSUE #55: Remove BlazeClient from dependencies
Provide a new API that works with any http4s server, such as Ember.

See https://github.com/trace4cats/trace4cats-cloudtrace/pull/72

### PR DESCRIPTION
**Pull request by catostrophe**
Closes #55

### CODE REVIEW

File: modules/newrelic-http-exporter/src/main/scala/io/janstenpickle/trace4cats/newrelic/Endpoint.scala [hunk 1]
```diff
@@ -1,16 +1,18 @@
 package io.janstenpickle.trace4cats.newrelic
 
+import org.http4s.Uri
+
 sealed trait Endpoint {
-  def url: String
+  def uri: Uri
```
Comment (catostrophe) [inline modules/newrelic-http-exporter/src/main/scala/io/janstenpickle/trace4cats/newrelic/Endpoint.scala:6]: This is a tiny but incompatible change. @janstenpickle don't you mind?

Comment (janstenpickle) [reply modules/newrelic-http-exporter/src/main/scala/io/janstenpickle/trace4cats/newrelic/Endpoint.scala:6]: I don't mid really. I'm just thinking we can maybe preserve the old API with a deprecation notice?

Comment (catostrophe) [reply modules/newrelic-http-exporter/src/main/scala/io/janstenpickle/trace4cats/newrelic/Endpoint.scala:6]: Sure, will rework

Comment (catostrophe) [reply modules/newrelic-http-exporter/src/main/scala/io/janstenpickle/trace4cats/newrelic/Endpoint.scala:6]: @janstenpickle we can preserve this:
```scala
  @deprecated
  def url: String = uri.renderString
```

Would you also like to keep the old constructor for `case class Observer(url: String)`?

**Review by janstenpickle — APPROVED** [review]
LGTM!

**Pull request merged**